tutor/templates/build/openedx/Dockerfile

```diff
@@ -159,7 +159,7 @@ USER ${APP_USER_ID}
 # https://hub.docker.com/r/powerman/dockerize/tags
 COPY --link --from=docker.io/powerman/dockerize:0.19.0 /usr/local/bin/dockerize /usr/local/bin/dockerize
 COPY --chown=app:app --from=edx-platform / /openedx/edx-platform
-COPY --chown=app:app --from=locales /openedx/locale /openedx/locale
+{% if not OPENEDX_ATLAS_PULL %}COPY --chown=app:app --from=locales /openedx/locale /openedx/locale{% endif %}
```

Comment:
Do we really need to disable `/openedx/locale`? It seems that in the `edxapp` both `atlas pull` and `/openedx/locale` can work together without an issue because it's depending on the `LOCALE_PATHS` variable which accepts multiple directories.

Reply:
Technically they can work together. Done!